test/integration/au/org/emii/aatams/ReceiverEventControllerTests.groovy

```diff
@@ -19,6 +25,6 @@ class ReceiverEventControllerTests extends AbstractControllerUnitTestCase
 
     void testExecuteReceiverEventByProject()
     {
-        assertExport([receiverDeployment: [station: [installation: [project: [eq: ["name", "Tuna"]]]]]], "testExecuteReceiverEventByProject")
+        assertExport([receiverDeployment: [station: [installation: [project: [in: ["name", "Tuna"]]]]]], "testExecuteReceiverEventByProject")
```

Comment:
Because "equality" filters are not implemented in the `QueryBuilder`.  It would've been pretty easy to do this, but more work than just changing the test (given that all text filters on the events page are autocomplete lists, rather than drop-down selects, i.e. equality filters are no used on the front-end).